pkg/testutil/fakes.go

```diff
@@ -498,3 +503,22 @@ func InvokeHTTPHandler(request *http.Request, path string, handler func(http.Res
 
 	return responseRecorder
 }
+
+//NewConfigMapLister creates a ConfigMapLister
+func NewConfigMapLister(kubeClient kubernetes.Interface) corelistersv1.ConfigMapLister {
+	return &simpleConfigMapLister{kubeClient: kubeClient}
+}
+
+//NewSecretLister creates a SecretLister
+func NewSecretLister(kubeClient kubernetes.Interface) corelistersv1.SecretLister {
+	return &simpleSecretLister{kubeClient: kubeClient}
+}
+
+//MockError mocks a fake kubernetes.Interface to return an expected error
```

Comment:
Great.  Thanks for adding this!

Reply:
Agree - this is exactly what I need for some tests I want to write :)